ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
```

Comment:
eslinting and stylinting will be added in big projects, so if we get used to do it now, will make our life simple in the future. Each css prop should be in alphabetical order